### DISCUSSION TITLE
Getting error message from parse when allow_exceptions is false

### DISCUSSION THREAD

**Discussion by lnmaurer** [nlohmann/json#3277] [Q&A] [answered]
When `allow_exceptions` in `parse` is set to `false`, is there any way to get the error message? Or is that only possible with exceptions? (I understand that, with `allow_exceptions` set to `false`, it's possible to test whether or not there was an error with `is_discarded`. I'm curious to know what the error was.)

**Comment by nlohmann** [comment, marked as answer]
This is only possible when you're using the SAX interface which passes an exception object, see https://json.nlohmann.me/api/json_sax/parse_error/.
